app/components/bali/calendar/previews/events.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Bali
+  module Calendar
+    module Previews
+      class Events
```

Comment:
Esta clase representa 1 solo evento, no múltiples eventos

```suggestion
      class Event
```